src/components/Swap/Settings/index.tsx

```diff
@@ -24,15 +24,11 @@ export function SettingsPopover() {
 }
 
 const SettingsButton = styled(IconButton)`
+  // don't rotate back when un-hovering because it works weirdly with modal backdrop opening on top
```

Comment:
be more specific here, so it's less likely someone will "unwind" this change

Reply:
also, use full sentence/grammar in your comment

Reply:
`// Don't rotate back when un-hovering so that clicking (and losing hover due to the modal backdrop) doesn't cause unintentional back-rotation.`?

Reply:
ugh the gen z in me is crying